toolkits/global/packages/global-loader-animation/scss/50-components/_loader.scss

```diff
@@ -0,0 +1,71 @@
+@keyframes lds-ripple {
+	0% {
+		top: 84px;
+		left: 84px;
+		width: 0;
+		height: 0;
+		opacity: 1;
+	}
+
+	100% {
+		top: 12px;
+		left: 12px;
+		width: 144px;
+		height: 144px;
+		opacity: 0;
+	}
+}
+
+@-webkit-keyframes lds-ripple {
+	0% {
+		top: 84px;
+		left: 84px;
+		width: 0;
+		height: 0;
+		opacity: 1;
+	}
+
+	100% {
+
+		top: 12px;
+		left: 12px;
+		width: 144px;
+		height: 144px;
+		opacity: 0;
+	}
+}
+
+.lds-ripple {
+	position: fixed;
+	width: 100px;
+	height: 100px;
+	left: 50%;
+	transform: translateX(-50%);
+	-webkit-transform: translate(-100%, -100%) scale(0.5) translate(50px, 50px);
+	transform: translate(-100%, -100%) scale(0.5) translate(50px, 50px);
+	z-index: $zindex-level-highest;
+
+	div {
```

Comment:
As this is global, we should have a `10-settings/_default.scss` file that sets some standard settings for this, then if this is SN a `10-settings/_springernature.scss` file that brands those setting for SN.

Candidates for abstracting to settings would be `border-width` and the two instances of `border-color`.

See an example of how it's done [in this component](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-card/scss/10-settings) with the brand settings importing default.